request/templatetags/request_tag.py

```diff
@@ -32,7 +32,6 @@ def render(self, context):
         return ''
 
 
-#@register.tag
```

Comment:
Now that we've dropped Python 2.3/4 support, we can probably just use the "new" `@` decorator syntax and drop the other lines `register.tag(active_users)`.

What do you think?